cmd/catchpointdump/net.go

```diff
@@ -203,15 +203,12 @@ func doDownloadCatchpoint(url string, wdReader util.WatchdogStreamReader, out io
 
 	for {
 		n, err := wdReader.Read(tempBytes)
-		if err == io.EOF {
-			return nil
-		}
-		if err != nil {
+		if err != nil && err != io.EOF {
```

Comment:
I assume this io.EOF is going to be caught again after Write() when you call Reset() later?